### PR TITLE
Add DF describe

### PR DESCRIPTION
**Pull request by cristineguadelupe**
Initial implementation of #157

### CODE REVIEW

**Review by philss — APPROVED** [review]
It looks good for the first version of `describe/1` :shipit: 

Congrats on your first contribution to Explorer! :tada: :smiley:

File: test/explorer/data_frame_test.exs [hunk 1]
```diff
@@ -2034,4 +2034,16 @@ defmodule Explorer.DataFrameTest do
                    fn -> DF.put(df, :e, Nx.tensor([1, 2, 3], type: {:u, 32})) end
     end
   end
+
+  test "describe/1" do
+    df = DF.new(a: ["d", "e", "f"], b: [1, 2, 3], c: ["a", "b", "c"])
```
Comment (josevalim) [inline test/explorer/data_frame_test.exs:2039]: Let's add a nil to one of the columns to see how it affects the result? In particular, I am wondering if count will decrease:

```suggestion
    df = DF.new(a: ["d", nil, "f"], b: [1, 2, 3], c: ["a", "b", "c"])
```

File: lib/explorer/data_frame.ex [hunk 2]
```diff
@@ -4389,6 +4389,31 @@ defmodule Explorer.DataFrame do
...
   end
 
+  @doc """
+  Describe numeric columns of a DataFrame.
+
+  Groups are ignored if the dataframe is using any.
+
+  ## Examples
+
+      iex> df = DF.new(a: ["d", "e", "f"], b: [1, 2, 3], c: ["a", "b", "c"])
```
Comment (josevalim) [inline lib/explorer/data_frame.ex:4399]:
```suggestion
      iex> df = DF.new(a: ["d", nil, "f"], b: [1, 2, 3], c: ["a", "b", "c"])
```

**Review by josevalim — COMMENTED** [review]
Woot! 🎉 Just one minor comment and we can ship it!

**Comment by josevalim** [comment]
@philss could you please open up an issue on polars and ask if they would also consider adding nulls count to describe?

**Comment by philss** [comment]
@josevalim they already had an issue about this, so I asked there: https://github.com/pola-rs/polars/issues/4068
Seems that the Python version shows this information.

**Pull request merged**